lib/analytics/course_students_csv_builder.rb

```diff
@@ -42,9 +47,21 @@ def row(courses_user)
     row << courses_user.character_sum_draft
     row << courses_user.references_count
     row << newbie?(courses_user.user)
+    row << total_articles_edited(courses_user.id)
+    row << total_articles_created(courses_user.id)
   end
 
   def newbie?(user)
     (@course.start..@course.end).cover? user.registered_at
   end
+
+  def total_articles_edited(id)
+    total_edits = @editors.count(id)
+    @editors.delete(id)
+    total_edits
+  end
+
+  def total_articles_created(id)
+    @new_articles_revisions.where(user_id: id).count
```

Comment:
This will count all new articles that the Dashboard knows about, not just ones created during the course.

Reply:
I thought it would only affect articles created in the course, because this query
`new_articles = @course.articles_courses.where(new_article: 1)` gets all the new articles within the course...

Then `@new_articles_revisions = Revision.where(article_id: new_articles)` uses the new_articles(course's articles) and ignores other articles.

Reply:
I think it will count all the revisions a user has made.. because I did not limit the `@new_articles_revisions` to search for `new_article` as well...

Once I correct the query above.. this should be fine..